### PR TITLE
openstack: Add RHEL 8.5 runner

### PR DESCRIPTION
**Pull request by henrywang**
Use " RHEL-8.5.0-x86_64-nightly-latest" image

### CODE REVIEW

**Review by atodorov — CHANGES_REQUESTED** [review]
Can we get the same configuration for aarch64 as well and aws as well if possible? 

Having this merged we should be able to switch all test runners for osbuild-composer instead of still keeping the old 8.4 ones!

**Comment by henrywang** [comment]
@atodorov Do we have rhel 8.5 image both for x86_64 and aarch64 on AWS?

**Comment by atodorov** [comment]
> @atodorov Do we have rhel 8.5 image both for x86_64 and aarch64 on AWS?

@ondrejbudai ^^^ how do we tell which AMI is which distro and more importantly do we have any 8.5 AMIs created ?

**Comment by ondrejbudai** [comment]
What do you mean by "which AMI is which distro"? For not yet released version of RHEL, I always did the following:

1) download a nightly EC2 image
2) unxz-it
3) upload it with `go run ./cmd/osbuild-upload-aws/` in `osbuild-composer` repository

**Comment by henrywang** [comment]
@atodorov Could you please help uploading rhel 8.5 image to AWS? I have AWS account for kernel testing, but it's not for osbuild group. I can add AWS configuration files in this PR. Thanks.

**Comment by atodorov** [comment]
ami-0da1ac7dca595deca - x86_64
ami-035a1bdfbb692f195 - aarch64

File: aws/rhel-8.5-aarch64/config.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "user": "ec2-user",
+    "runnerArch": "aarch64",
+    "prepareScript": "sudo tee /etc/yum.repos.d/redhat.repo <<EOT\n[RHEL-8-NIGHTLY-BaseOS]\nname=baseos\nbaseurl=http://download.devel.redhat.com/rhel-8/nightly/RHEL-8/latest-RHEL-8.5.0/compose/BaseOS/aarch64/os/\nenabled=1\ngpgcheck=0\n\n[RHEL-8-NIGHTLY-AppStream]\nname=appstream\nbaseurl=http://download.devel.redhat.com/rhel-8/nightly/RHEL-8/latest-RHEL-8.5.0/compose/AppStream/aarch64/os/\nenabled=1\ngpgcheck=0\n\n[RHEL-8-NIGHTLY-CRB]\nname=crb\nbaseurl=http://download.devel.redhat.com/rhel-8/nightly/RHEL-8/latest-RHEL-8.5.0/compose/CRB/aarch64/os/\nenabled=1\ngpgcheck=0\nEOT"
```
Comment (atodorov) [inline aws/rhel-8.5-aarch64/config.json:4]: Please rebase to `main` and rename the repository file to `rhel8internal.repo`. See the last commit for details why.

**Review by atodorov — CHANGES_REQUESTED** [review]

File: aws/rhel-8.5-aarch64/config.json (hunk 1 above)
Comment (henrywang) [reply aws/rhel-8.5-aarch64/config.json]: Updated.

**Review by atodorov — APPROVED** [review]

**Pull request merged**